AsyncDisplayKitTests/ASTextNodeSnapshotTests.m

```diff
@@ -25,8 +24,9 @@ - (void)testTextContainerInset
   ASTextNode *textNode = [[ASTextNode alloc] init];
   textNode.attributedText = [[NSAttributedString alloc] initWithString:@"judar"
                                                             attributes:@{NSFontAttributeName : [UIFont italicSystemFontOfSize:24]}];
-  [textNode layoutThatFits:ASSizeRangeMake(CGSizeZero, CGSizeMake(CGFLOAT_MAX, CGFLOAT_MAX))];
   textNode.textContainerInset = UIEdgeInsetsMake(0, 2, 0, 2);
+  ASLayout *layout = [textNode layoutThatFits:ASSizeRangeMake(CGSizeZero, CGSizeMake(CGFLOAT_MAX, CGFLOAT_MAX))];
+  textNode.frame = CGRectMake(0, 0, layout.size.width, layout.size.height);
```

Comment:
We can use it in this case but not in the other two as we set a specific position != `CGPointZero`